tests/integration/_common.py

```diff
@@ -60,8 +60,12 @@ def retrieve_fixture_for_step(
         The specific test step fixture
     """
     fixture_path = fixture_path_from_request(request, step_index, test_name)
+    expected_output = []
+
     with fixture_path.open(encoding="utf-8") as fh:
-        expected_output = json.load(fh)["output"]
+        data = json.load(fh)
```

Comment:
this is not necessary as the output key will always be present b/c the fixtures are programatically generated and not manually maintained

Reply:
This is needed for type safety, logic does not change.